sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/statusdisplay/State.groovy

```diff
@@ -0,0 +1,33 @@
+package life.qbic.portal.sampletracking.components.projectoverview.statusdisplay
+
+
+/**
+ * <b>The state of a project status.</b>
+ *
+ * <p>A status can have the following states:
+ * <ul>
+ *     <li><b>COMPLETED</b>: When all samples in a project have reached this status and the status is not a failure status.</li>
+ *     <li><b>FAILED</b>: When a status is a failure status and samples are presently in the failed status.</li>
+ *     <li><b>IN_PROGRESS</b>: When not all samples were evaluated for the current status. <b>OR</b>: If the status is a failure status and there are not samples in the status.</li>
```

Comment:
I'm confused by the failure status in this description. 
I'd argue that if there is no sample in the failure status then the column should be set to COMPLETED and not IN_PROGRESS.

Reply:
This is not possible thus we decided against that.